src/sms-password/templates/user_management/sms-password/reset_form.html

```diff
@@ -6,35 +6,36 @@
     <p translate>You've successfully changed your password.</p>
 </div>
 
-<div class="form-group" ng-class="{'has-error': errors.phone_number}">
+<div class="form-group" ng-class="{'has-error': errors.phone_number || errors.non_field_errors}">
     <label for="phone_number" ng-show="fields.phone_number.label" ng-bind="fields.phone_number.label"></label>
     <input type="text" ng-model="data.phone_number" class="form-control" id="phone_number" autofocus>
     <div class="help-block" ng-show="errors.phone_number" ng-bind="errors.phone_number"></div>
+    <div class="help-block" ng-if="errors.non_field_errors" ng-bind="errors.non_field_errors"></div>
```

Comment:
I can see in the diff that it was on it's own next to `errors.detail`, not in the phone number field.